.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: 'elixir'
 
-elixir: 1.8.1
-otp_release: 21.3
+elixir: 1.10.2
+otp_release: 22.0.7
```

Comment:
Since we are updating to Elixir `1.10`, should we update to the latest Erlang `22.2`?

Reply:
I tried it and the build fails:

![image](https://user-images.githubusercontent.com/464900/76471257-91dcd800-63c8-11ea-9692-ce9632941886.png)

Reply:
This could be updated in another PR